doc/src/devdocs/build/build.md

```diff
@@ -269,3 +269,16 @@ If you need to build Julia from source with a Git checkout of a stdlib, then use
 For example, if you need to build Julia from source with a Git checkout of Pkg, then use `make DEPS_GIT=Pkg` when building Julia. The `Pkg` repo is in `stdlib/Pkg`, and created initially with a detached `HEAD`. If you're doing this from a pre-existing Julia repository, you may need to `make clean` beforehand.
 
 If you need to build Julia from source with Git checkouts of more than one stdlib, then `DEPS_GIT` should be a space-separated list of the stdlib names. For example, if you need to build Julia from source with a Git checkout of Pkg, Tar, and Downloads, then use `make DEPS_GIT='Pkg Tar Downloads'` when building Julia.
+
+## Building an "assert build" of Julia
+
+An "assert build" of Julia has both `FORCE_ASSERTIONS=1` and `LLVM_ASSERTIONS=1`. To build
+an assert build, you can run the following commands:
+
+```
+git clone https://github.com/JuliaLang/julia.git
+cd julia
+make FORCE_ASSERTIONS=1 LLVM_ASSERTIONS=1
```

Comment:
I normally recommend setting them in Make.user so that when you recompile you don't suddenly lose the assert nature.

Reply:
Good call. I'll make that change.

Reply:
Done! @vchuravy can you double check that the syntax is correct?